src/index.html

```diff
@@ -28,6 +28,14 @@ <h2>Add you score</h2>
           <input type="number" placeholder="Your score" id="score" required />
           <button type="submit" class="add-btn btn">Submit</button>
         </form>
+        <div class="status">
+          <div class="success hidden">
+            <span>Your score has been added successfully. ✅ 🎯 </span>
+          </div>
+          <div class="error hidden">
+            <span>Something went wrong, please try again. 🚫 </span>
```

Comment:
- [optional] wonderful job adding an error message for wrong submission. Please, I advise you add more styles, even a nice background will make your design more appealing. Nice work.